3기/LeeGaYeon/week1/page1/style.css

```
@@ -0,0 +1,87 @@
+body {
+  /* font */
+  font-family: Spoqa Han Sans Neo;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 25px;
+  letter-spacing: 0em;
+  text-align: left;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  height: 100vh;
+  gap: 16px;
+}
+
+header,
+footer,
+aside,
+main {
+  background-color: #d9d9d9;
+}
+
+header {
+  max-width: 960px;
+  width: 100%;
+  height: 100px;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 16px;
+  max-width: 960px;
+  max-height: 668px;
+  height: 100%;
+  width: 100%;
+}
+
+main {
+  grid-column: span 3;
+  display: grid;
+  gap: 16px;
+  padding: 32px;
+  max-width: 744px;
+}
```

그리드를 사용한게 틀린것은 아니예요!
flexbox를 사용하셔도 좋고 grid를 사용하셔도 좋은데,
지금같은 경우는 아래처럼 사용하면 더 좋을 것 같아요.
```suggestion
.container {
  display: grid;
  grid-template-columns: 200px 1fr;
  gap: 16px;
  max-width: 960px;
  max-height: 668px;
  height: 100%;
  width: 100%;
}

main {
  display: grid;
  gap: 16px;
  padding: 32px;
  max-width: 744px;
}
```

`.container`에서 `grid-template-columns` 값을 `200px 1fr`로 변경해서
"사이드바"의 크기는 200px으로 고정했고,
main의 크기는 1fr로 나머지 영역을 모두 가져가게 했어요.
(라이브세션때 "어느 영역이 줄어드는지 파악하는게 중요하다"라고 말씀드린게 이런 이유입니다. 😊)

이렇게 하면

`main` CSS에서 `grid-column: span 3;` 속성은 추가할 필요가 없어지기 때문에 제거할 수 있게 되어요.

이렇게 굳이 들어가지 않아도 되는 속성을 제거하다보면 반응형 스타일을 작업할때도 수월해질거예요!